src/eredis.erl

```diff
@@ -27,15 +27,52 @@
 %% Exported for testing
 -export([create_multibulk/1]).
 
+-type transport() :: tcp | ssl.
 -export_type([transport/0]).
+
+-type reconnect_sleep() :: no_reconnect | integer().
+
+-type option() ::
+        {transport, transport()} |
+        {host, string()} |
+        {host, {local, term()}} |
+        {port, integer()} |
+        {database, string()} |
+        {database, undefined} |
+        {password, string()} |
+        {reconnect_sleep, reconnect_sleep()} |
+        {connect_timeout, non_neg_integer()}.
+-type server_args() :: [option()].
+
+-type sub_option() ::
+        option() |
+        {max_queue_size, non_neg_integer() | infinity} |
+        {queue_behaviour, drop | exit}.
+-type sub_args() :: [sub_option()].
+
+-type return_value() :: undefined | binary() | [binary() | nonempty_list()].
 -export_type([return_value/0]).
 
+-type pipeline() :: [iolist()].
+-export_type([pipeline/0]).
+
+-type channel() :: binary().
```

Comment:
If this is part of the API, we should export it.

Reply:
Perhaps you fixed it later? Please resolve if that's the case. I'm sorry, I don't think this commit-by-commit approach to reviews works very well for me...

Reply:
It is exported.